CHANGELOG.md

```diff
@@ -56,6 +56,8 @@ For upgrade instructions read the [UPDATING](UPDATING.md) guide.
   was raised to 100000. #493
 - `Value::from` is now implemented for `Error` as public API to create invalid values.
   Previously this behavior was hidden internally in the serde support. #495
+- `UndefinedBehavior::Strict` now acts more delayed.  This means that now `value.key is defined`
+  will no longer error when `value.key` is `undefined`. #500
```

Comment:
Isn't that a little wrong.

```diff
- `value.key` is `undefined`
+ `value` does not have a member `key`.
```
or similarly.

Reply:
I fixed the sentence in the changelog: 

> `UndefinedBehavior::Strict` now acts more delayed.  This means that now `value.key is defined` will no longer fail.